pages/ksphere/kommander/1.0/release-notes/index.md

```diff
@@ -27,6 +27,9 @@ enterprise: false
 
 - Add 1.16.8 to selectable Kubernetes versions.
 
+#### Known issues
+
+- <strong>Kommander cluster creation before Konvoy v1.4.3: </strong>In versions before Konvoy v1.4.3, with Kommander version v1.0, new Kommander clusters cannot be created. It is recommended you upgrade to the latest version of Konvoy (v1.5.0), which includes Kommander 1.1.0 to be able to create new Kommander clusters.
```

Comment:
@samvantran - 
So, I did check this out today - and there's something that's cooked into Konvoy when you deploy Kommander, where if you deploy v 1.1.0 Kommander straight up to a cluster running konvoy 1.4.0, it gets weird. Like, you can't add clusters, or infrastructure providers. So, it looks like you'd still need to upgrade to a newer version of Konvoy regardless. :-|

Reply:
Yup, that is because Komm1.1x is not supported on Konvoy 1.4.x https://docs.d2iq.com/ksphere/konvoy/1.5/release-notes/#disclaimer-2

This disclaimer is on the 1.5 release note BUT it looks like they're missing from the 1.4 docs which is not good. I remember it being backported but maybe it got removed. Anyway, let's get those same disclaimers into the 1.4 release notes.

Reply:
Let's keep your phrasing and not use my suggestion since it leaves room for interpretation that you can use 1.1x on 1.4

Reply:
K, watch for commit, @samvantran